app/assets/js/generator.js

```diff
@@ -19,36 +19,52 @@
     var tpl = doT.template($('#plugins-all-template').text());
     var pluginsAll = $('#plugins-all');
 
-    $.getJSON('https://storage.googleapis.com/generators.yeoman.io/cache.json')
-    .done(function (plugins) {
-      if (Array.isArray(plugins)) {
-        pluginsAll.html(tpl({
-          modules: plugins.sort(function (a, b) {
-            return a.stars === b.stars ? 0 : a.stars < b.stars ? 1 : -1;
-          })
-        }));
-
-        var list = new win.List('plugins-all', {
-          valueNames: [
-            'name',
-            'owner',
-            'stars',
-            'updated',
-            'downloads',
-            'description'
-          ]
-        });
-
-        if (list.listContainer) {
-          list.on('updated', function () {
-            $('.table thead').toggle(list.matchingItems.length !== 0);
-            $('#search-notfound').toggle(list.matchingItems.length === 0);
+    var lastSha = win.localStorage.getItem('sha');
```

Comment:
I know localStorage is super cool and all, but IMO that could be done pretty easily with a cookie. You'd have 100% browser support + automatic cache cleaning.